doc/cmp.txt

```diff
@@ -730,8 +730,9 @@ window.documentation.max_height~
 
                                             *cmp-config.experimental.ghost_text*
 experimental.ghost_text~
-  `boolean | { hl_group = string }`
+  `boolean | { hl_group = string inline_emulation = boolean}`
   Whether to enable the ghost_text feature.
+  inline_emulation is emulating inline ghost text for older neovim versions
```

Comment:
Please make punctuation properly.

Which neovim version are you talking about --- that doesn't support inline text? This should be explicitly mentioned.

Reply:
> Please make punctuation properly.
Sorry.

> Which neovim version are you talking about --- that doesn't support inline text? This should be explicitly mentioned.

v0.8.0+